sql/base/dev/world_database.sql

```diff
@@ -3829,15 +3829,14 @@ CREATE TABLE `warden_checks` (
   `type` tinyint(3) UNSIGNED NULL DEFAULT NULL,
   `flags` int(10) NOT NULL DEFAULT 0,
   `enabled` tinyint(1) NOT NULL DEFAULT 0,
-  `platform` varchar(5) NOT NULL,
   `data0` varchar(255) DEFAULT NULL,
   `data1` varchar(255) DEFAULT NULL,
   `address` int(10) UNSIGNED NULL DEFAULT NULL,
   `length` tinyint(3) UNSIGNED NULL DEFAULT NULL,
   `result` varchar(24) DEFAULT NULL,
   `comment` text DEFAULT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=MyISAM AUTO_INCREMENT=787 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
```

Comment:
can we stick to utf8mb4 ? :)

Reply:
Sure, whatever, that's the rebase. Thought I'd update the branch, don't really mean to finish it yet.

Reply:
yeah, no rush
just want to keep the proper utf8 here